src/custom/pages/Swap/index.tsx

```diff
@@ -78,68 +74,28 @@ const SwapModWrapper = styled(SwapMod)`
       color: ${({ theme }) => theme.text1};
     }
 
-    ${ArrowWrapperUni} {
```

Comment:
should this be commented out no? or is this our code we added

Reply:
Our code we added (also in relation to the import comment above).

Reply:
This is the index file, not the Mod, so I rather keep it minimal than having commented out code.
Or we do that for index as well and I have been doing this wrong all along? :scream:

Reply:
no no i didnt see the custom in the path my B yo